WordPress/gutenberg
# viewport: set up autogenerated API docs in README

**Related to** https://github.com/WordPress/gutenberg/issues/14227

This PR sets up auto-generated API docs for the `viewport` package through the `npm run docs:build` command and updates its README file.

## Test instructions

- Delete or modify the contents at `packages/viewport/README.md` within the start `` and end `` tokens.
- Execute `npm run docs:build` and verify contents are re-created.
- Review that the API docs match the JSDoc of the exported symbols.
